dojo/importers/reimporter/reimporter.py

```diff
@@ -105,258 +114,244 @@ def process_parsed_findings(
                             "{}".format(err)
                         )
 
-            item.hash_code = item.compute_hash_code()
-            deduplicationLogger.debug("item's hash_code: %s", item.hash_code)
+            reimported_finding.hash_code = reimported_finding.compute_hash_code()
+            deduplicationLogger.debug("item's hash_code: %s", reimported_finding.hash_code)
 
-            findings = reimporter_utils.match_new_finding_to_existing_finding(
-                item, test, deduplication_algorithm
+            existing_findings = reimporter_utils.match_new_finding_to_existing_finding(
+                reimported_finding, test, deduplication_algorithm
             )
 
             deduplicationLogger.debug(
-                "found %i findings matching with current new finding", len(findings)
+                "found %i findings matching with current new finding", len(existing_findings)
             )
 
-            if findings:
-                # existing finding found
-                finding = findings[0]
-                if finding.false_p or finding.out_of_scope or finding.risk_accepted:
-                    logger.debug(
-                        "%i: skipping existing finding (it is marked as false positive:%s and/or out of scope:%s or is a risk accepted:%s): %i:%s:%s:%s",
-                        i,
-                        finding.false_p,
-                        finding.out_of_scope,
-                        finding.risk_accepted,
-                        finding.id,
-                        finding,
-                        finding.component_name,
-                        finding.component_version,
-                    )
-                    if (
-                        finding.false_p == item.false_p
-                        and finding.out_of_scope == item.out_of_scope
-                        and finding.risk_accepted == item.risk_accepted
-                    ):
-                        unchanged_items.append(finding)
-                        unchanged_count += 1
-                        continue
-                elif finding.is_mitigated:
+            if existing_findings:
+                # existing findings found
+                existing_finding = existing_findings[0]
```

Comment:
Just wondering if we should log something here for a warning... or if a loop is needed to process all the matched existing findings instead of just the first one ?

Reply:
Yes I was wondering something similar... For now we could write a debug/log statement. The purpose of this pull request was not to significantly change the behaviour of the reimporter (which this might do). Although I agree that probably this case should be handled somehow, I think this should be handled in a seperate ticket/issue.